pro_tes/ga4gh/tes/task_runs.py

```diff
@@ -213,6 +209,11 @@ def create_task(  # pylint: disable=too-many-statements,too-many-branches
             )
             return {"id": db_document.task.id}
 
+        # set TES state to SYSTEM_ERROR as task submission failed on all the available TES instances.
```

Comment:
This line is too long, lines should be <80 chars.

Reply:
Okay. I have made the required changes in the next commit